subsys/zigbee/cli/zigbee_cli.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <init.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+#include <shell/shell_rtt.h>
+
+#include <zb_nrf_platform.h>
+#include "zigbee_cli.h"
+
+/* Endpoint used by the Zigbee CLI. */
+static zb_uint8_t cli_ep;
+
+/* Zigbee CLI debug mode indicator. */
+static zb_bool_t debug_mode = ZB_FALSE;
+
+LOG_MODULE_REGISTER(cli, CONFIG_ZIGBEE_CLI_LOG_LEVEL);
+
+
+/**@brief Function for CLI shell initial configuration.
+ *        To be called by SYS_INIT.
+ *
+ * @param[in] unused Unused param.
+ *
+ * @returns By default returns 0.
+ */
+static int zb_cli_init(struct device *unused)
```

Comment:
`zbi_shell_init`

Reply:
`zb_shell_init` or `zb_cli_shell_init`? To remain consistency `zb_cli` shouldn't be dropped.

Reply:
I am not sure about this. In Zephyr, there is no `cli` and all modules that use console are having `shell` in their name. For example: `bt_shell`, `flash_shell`, `adc_shell`, `wifi_shell` or `telnet_shell`.
In Zephyr / NCS `cli` is not used at all, it is legacy naming from our SDK.

If you are in rush it can stay but in my opinion, we shall spend some time and rename each `cli` to `shell`. It could be done with some python script.

Reply:
@jakub-uC I had a discussion about this with @sebastiandraus. This PR is dragging, so we decided to keep as is for now. We're going to port more CLI commands. How about we rename the module name and Kconfig option to ```ZIGBEE_SHELL``` now but rename any non-user facing functions is a later PR?

Reply:
We can proceed like that - thank you